userChrome.css

```diff
@@ -9,14 +9,14 @@
   Description:    Minimalist, Simple, Keyboard Centered and based on SimpleFox. 🦊
                   What you get is a really simple responsive one-line layout using the new Proton UI.
 
-                  > SimpleFox: https://github.com/migueravila/SimpleFox
+                  Based on SimpleFox: https://github.com/migueravila/SimpleFox
```

Comment:
It's stated above in the Description that it's based on SimpleFox, thus I don't think it's needed to be mentioned here again. Even tho it's not much, this makes it a tiny bit more convoluted.

Reply:
> convoluted

may i know how?
i mean i understand, but at first sight when seeing it it's not very obvious.

Reply:
HOW DID I NOT SEE THAT WHAT

Reply:
> HOW DID I NOT SEE THAT WHAT

Well, i guess that proves my second thought.. that while I think it's obvious, to someone just skimming through it might not be. So this probably isn't a bad change after all? :D

Reply:
AAAAAAAA JUST TELL ME IF I SHOULD KEEP IT OR NOT

Reply:
Sorry. ,_,
I still dislike that the »based on« is doubled, I'm thinking about how to change the description text above to have the best of both worlds. So keep it, and I'll dop the description when I come up with something in a bit? c:

Reply:
actually, thats a much better idea.

~~i did not like the description anyway.~~

off-topic question:
why is the userChrome.css file structured so weirdly? (eg. identation, giant clarifiers like CONFIG and COLOURS, ~~tons of whitespace, could be resolved by just simply making the file more readable?~~)